index.md

```diff
@@ -128,7 +117,7 @@ ms.assetid:
                                         <div class="card">
                                             <div class="cardImageOuter">
                                                 <div class="cardImage">
-                                                    <img src="./docs/images/hub/net-gs-3.svg" alt="" />
+                                                    <img src="articles/images/hub/logo_NETframework.svg" alt="" />
```

Comment:
I see the issue. The hover image in the ad cards was pointing to the wrong place; I've made these consistent across the board.